lib/presentation/view/modal_sheets/edit_text_modal_sheet_builder.dart

```diff
@@ -170,12 +161,12 @@ class EditTextModalSheetBuilder {
                         mainAxisAlignment: MainAxisAlignment.end,
                         children: [
                           TextButton(
-                            onPressed: () => {Navigator.pop(context), _debounce?.cancel(), _onCancelActionClick()},
+                            onPressed: () => {Navigator.pop(context), _debounce?.cancel()},
                             child: Text(_cancelText.toUpperCase(),
                                 style: TextStyle(color: AppColor.primaryColor)),
                           ),
                           TextButton(
-                            onPressed: () => (_error == null || _error.isEmpty) ? _onConfirmActionClick(_textController.text) : null,
+                            onPressed: () => (_error == null || _error.isEmpty) ? {Navigator.pop(context), _onConfirmActionClick(_textController.text)} : null,
```

Comment:
why not split to a method? Too difficult to understand. srr, my stupid brain

Reply:
oke boy